lightning/restore/restore.go

```diff
@@ -672,8 +671,30 @@ var gcLifeTimeKey struct{}
 func (rc *RestoreController) restoreTables(ctx context.Context) error {
 	logTask := log.L().Begin(zap.InfoLevel, "restore all tables data")
 
-	var wg sync.WaitGroup
+	// for importer/local backend, we should disable some pd scheduler and change some settings, to
+	// make split region and ingest sst more stable
+	if rc.cfg.TikvImporter.Backend != config.BackendTiDB {
```

Comment:
How about make another config to control the schedulers?

Reply:
Are there any situations we want to use local/importer backend but still want to enable these schedulers? Since they have big impacts, I think we should always disable them in any case? In cases when we should disable the schedulers, maybe tidb backend is a proper choice then

Reply:
ok, another problem is when lightning exits abnormally after remove these schedulers. next time lightning will reset nothing even it successfully restore data without any errors. so we should follow this https://github.com/tikv/pd/issues/2782, and better to have a way to set the lost config/schedulers back.